### PR TITLE
[Feature] Change the placeholder in the input bar

### PR DESCRIPTION
**Pull request by KaterinaWire**
## What's new in this PR?

If the contact has a set availability status and a long name, the placeholder in the input bar can be broken for WEB app. It was decided for all platforms to show "This person is" + availability status.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Availability/AvailabilityStringBuilder.swift [hunk 1]
```diff
@@ -44,7 +44,7 @@
                 guard availability != .none && !user.shouldHideAvailability else { //Should use the default placeholder string
                     return nil
                 }
-                title = "availability.\(availability.canonicalName).placeholder".localized(args: user.name ?? "").localizedUppercase
+                title = "availability.\(availability.canonicalName).placeholder".localized(args: "availability.default.user_name.title".localized).localizedUppercase
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Availability/AvailabilityStringBuilder.swift]: I'd suggest placing the "This person" in all the existing placer holder strings, like this:
`"availability.available.placeholder" = "This person is available";`

The reason is that it's problematic to assume that we can concatenate the two strings "This person" and "is available" in all languages.

File: Wire-iOS/Sources/UserInterface/Availability/AvailabilityTitleView.swift [hunk 2]
```diff
@@ -120,7 +120,7 @@ final class AvailabilityTitleView: TitleView, Themeable, ZMUserObserver {
         super.configure(icon: icon, title: title, interactive: isInteractive, showInteractiveIcon: showInteractiveIcon)
         
         accessibilityLabel = options.contains(.allowSettingStatus) ? "availability.accessibility_label.change_status".localized : "availability.accessibility_label.status".localized
-        accessibilityValue = availability.localizedName
+        accessibilityValue = availability.localizedName + " " + (user.name ?? "")
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Availability/AvailabilityTitleView.swift]: Why is this necessary now? You were modifying the placeholder in the input bar and this will affect the navigation title.

`accessibilityValue` is user facing for people who rely on voice over.

Comment (KaterinaWire) [reply Wire-iOS/Sources/UserInterface/Availability/AvailabilityTitleView.swift]: Removed
